classes/class-wc-connect-shipping-label.php

```diff
@@ -342,8 +341,12 @@ public function is_order_eligible_for_shipping_label_creation( WC_Order $order )
 			// Update the quantity for each refunded product ID in the order.
 			foreach ( $order->get_refunds() as $refund ) {
 				foreach ( $refund->get_items() as $refunded_item ) {
-					$product    = WC_Connect_Compatibility::instance()->get_item_product( $order, $refunded_item );
-					$product_id = WC_Connect_Compatibility::instance()->get_product_id( $product );
+					$product = WC_Connect_Compatibility::instance()->get_item_product( $order, $refunded_item );
```

Comment:
The `WC_Connect_Compatibility::instance()->get_item_product(..)` shares the same code between the 2 compatibility classes. It would be better to move this out from compatibility, maybe into a util function or something.